packages/synapse-react-client/src/components/Markdown/MarkdownSynapse.test.tsx

```diff
@@ -305,7 +305,8 @@ describe('MarkdownSynapse tests', () => {
     expect(spyOnRender).toHaveBeenCalled()
   })
 
-  it('renders a synapse reference', async () => {
+  // PORTALS-3471:  The migration to DomPurify broke the markdown reference/bookmark functionality
+  it.skip('renders a synapse reference', async () => {
```

Comment:
Do you know if this is a limitation in DomPurify? Or does this require additional investigation (a new ticket)? This is used in at least one portals page: https://nf.synapse.org/Explore/Hackathon/DetailsPage/Background%24Results?id=syn24304262

Reply:
Fixed!  The order of the attributes changed, so the regex to find these elements also needed to be updated (to no longer assume the order)